test/gui/tst_syncing/test.feature

```diff
@@ -6,7 +6,26 @@ Feature: Syncing files
 
     Scenario: Syncing a file to the server
         Given user "Alice" has been created on the server with default attributes
-        And user "Alice" has set up a client with default settings and polling interval "5000"
+        And user "Alice" has set up a client with these settings and password "1234":
```

Comment:
Why is this change required. Same goes for other changes down below.

Reply:
It must have been changed while I rebased it. I will change it back as it is in master.